src/main/resources/org/jenkinsci/plugins/workflow/cps/steps/LoadFromClasspathStep/config.jelly

```diff
@@ -0,0 +1,20 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+
+	<f:entry field="path" title="${%Classpath Resource Path}" description="The classpath location of the Groovy Pipeline Script to load">
+	    <f:textbox/>
+	</f:entry>
+	
+	<f:entry field="classloaderSource" title="${%Classpath Hint Object}" description="An object to grab a ClassLoader from to use to load the script">
+	    <f:textbox default="null"/>
```

Comment:
I would like the name the developer types into this box to be used in the snippet generator exactly as-is, _without_ surrounding it with quotation marks, so that `this` or `someVariableThatWillExistInTheirPipeline` can be used.

I haven't been able to figure this out - is it possible with the stapler-powered snippet generator? If so, what do I need to do?